helix-term/src/commands.rs

```diff
@@ -1764,6 +1766,14 @@ fn page_cursor_half_down(cx: &mut Context) {
     scroll(cx, offset, Direction::Forward, true);
 }
 
+fn scroll_line_up(cx: &mut Context) {
+    scroll(cx, 1, Direction::Backward, false);
+}
+
+fn scroll_line_down(cx: &mut Context) {
+    scroll(cx, 1, Direction::Forward, false);
```

Comment:
Instead of hard-coding `1` let's make this `cx.count()` so that you can move 3 lines up with `3<C-y>` for example

Reply:
Thanks for the suggestion. Done.